src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  // The car coste calculator
+  let result = days * 40;
+
+  if (days >= '7') {
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md): Don't use literals in expressions (HARDCODE). Instead use constants explaining this values